com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.adamcake.Bolt
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install . --prefix=${FLATPAK_DEST}
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
this is not allowed on flathub and the build will fail. It must be done offline.

Reply:
Not sure what to do then, I can't find any way to get pip to operate without a network connection other than installing a .whl from here, but in that case I'd have to have four different cases for x86_66/aarch64 and glibc/musl, which defeats the point of why I used Python for this.

Reply:
Maybe resolved in 7bc9a91?